### PR TITLE
task 58 solved

### PR DESCRIPTION
**Pull request by arthur-borisenko**

### CODE REVIEW

File: week3/58-standartnoie-otklonieniie/task.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    elements_count = 0
+    for el in seq:
+        elements_count += 1
+        elements_sum += el
+    res = elements_sum / elements_count
+    return res
+
+
+def get_sequence() -> list:
+    seq = []
```
Comment (IgorNB) [inline week3/58-standartnoie-otklonieniie/task.py:15]: can you use O(1) memory instead of O(n)?

**Review by IgorNB — CHANGES_REQUESTED** [review]

File: week3/58-standartnoie-otklonieniie/task.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    def __next__(self):
+        self.current = int(input())
+        if self.current != 0:
+            return self.current
+        raise StopIteration
+
+
+def main():
+    input_stream = InputIntStream()
+    elements_sum = 0
```
Comment (IgorNB) [inline week3/58-standartnoie-otklonieniie/task.py:20]: provide method with meaningfull name.

File: week3/58-standartnoie-otklonieniie/task.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    elements_count = 0
+    elements_squares_sum = 0
+    for el in input_stream:
+        elements_count += 1
+        elements_sum += el
+        elements_squares_sum += el ** 2
+    average = elements_sum / elements_count
+    result = math.sqrt((elements_squares_sum -
```
Comment (IgorNB) [inline week3/58-standartnoie-otklonieniie/task.py:28]: add comment explaning how formula from task was converted to this formula

**Review by IgorNB — APPROVED** [review]

File: week3/58-standartnoie-otklonieniie/task.py (hunk 2 above)
Comment (IgorNB) [reply week3/58-standartnoie-otklonieniie/task.py:20]: critical. make method reusable

**Review by IgorNB — CHANGES_REQUESTED** [review]

File: week3/58-standartnoie-otklonieniie/task.py (hunk 3 above)
Comment (IgorNB) [reply week3/58-standartnoie-otklonieniie/task.py:28]: critical

**Review by IgorNB — CHANGES_REQUESTED** [review]

File: week3/58-standartnoie-otklonieniie/task.py (hunk 3 above)
Comment (arthur-borisenko) [reply week3/58-standartnoie-otklonieniie/task.py:32]: done

Comment (arthur-borisenko) [reply week3/58-standartnoie-otklonieniie/task.py:32]: done

**Review by IgorNB — APPROVED** [review]

**Pull request merged**